test/libsolidity/util/TestFileParser.cpp

```diff
@@ -158,6 +166,24 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 	return calls;
 }
 
+void TestFileParser::checkBuiltinFunction(std::string const& signature)
+{
+	assert(m_builtins != nullptr);
+	vector<string> builtinPath;
+	boost::split(builtinPath, signature, boost::is_any_of("."));
+	assert(builtinPath.size() == 2);
+
+	auto module = m_builtins->find(builtinPath.front());
+	if (module == m_builtins->end())
+		throw TestParserError("builtin module '" + builtinPath.front() + "' not found");
+
+	auto builtin = module->second.find(builtinPath.back());
+	if (builtin == module->second.end())
+		throw TestParserError(
```

Comment:
Please try to adhere to the coding style. Also, plain `throw` are discouranged, please use the boost macro that adds file and line information.

Reply:
I used the same structure for the exceptions as present in this file. Should I replace all of them to use the boost macro?

Reply:
yes, sounds good.